hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveClientPool.java

```diff
@@ -44,7 +44,7 @@ public void testConf() {
     conf.set(HiveConf.ConfVars.METASTOREWAREHOUSE.varname, "file:/mywarehouse/");
     conf.setInt("iceberg.hive.client-pool-size", 10);
 
-    HiveClientPool clientPool = new HiveClientPool(conf);
+    HiveClientPool clientPool = new HiveClientPool(10, conf);
```

Comment:
Do we need 10? What was the original value?

Reply:
Previously we had a HiveClientPool constructor, which got the client-pool-size from the hive conf. This was removed, and now it expects the pool-size explicitly.